install_config/aggregate_logging.adoc

```diff
@@ -1974,6 +1976,14 @@ $ curl --key /etc/elasticsearch/secret/admin-key \
   "https://localhost:9200/project.logging.3b3594fa-2ccd-11e6-acb7-0eb6b35eaee3.2016.06.15"
 ----
 
+[[fluentd-redeploy-certs]]
+== Redeploy EFK Certificates
+
+If you need to redeploy the EFK certificates, first remove the certificates from the `{/location/of/your/base/ocp/install}/logging` directory 
+on your the first master listed in the Ansible host inventory. Then re-run the logging installer. 
+Make sure the xref:logging-custom-certificates[Custom Certificate] parameters are set 
+in your xref:../install/configuring_inventory_file.adoc#configuring-ansible[inventory host file].
+
```

Comment:
@ewolinetz Do we have a default location for the certificate directory, or something more real-world than `location/of/your/base/ocp/install/logging`?

Reply:
@mburke5678 do we have a defined location in our docs of where the master/node configs are installed? it is the same base path as that (and it is different for an origin and enterprise install)

Reply:
@mburke5678 No,  Redeploy the pod is enough.( the pod wasn't restarted as the step 2 failed. so we need to redeploy pod manually)